data/json/recipes/weapon/explosive.json

```diff
@@ -63,7 +63,7 @@
     "autolearn": true,
     "qualities" : [ { "id": "CUT", "level": 1 } ],
     "components": [
-      [ [ "thread", 10 ] ],
+      [ [ "filament", 10, "LIST" ] ],
```

Comment:
I believe this part was suppose to represent the fuse. Does specifically the sinew burn like this?

Reply:
Sinews are organic, so they certainly burn, but not as good as threads.

Reply:
Good call, in reality even the string is pushing it, in reality fuses are impregnated with gunpowder or other compounds to increase burning temperature, an ordinary cotton string would be slow and unreliable.